source/release-notes.txt

```diff
@@ -10,6 +10,24 @@ Release Notes
    :depth: 1
    :class: twocols
 
+1.3.2 Changelog
+---------------
+
+*Released October 25, 2023*
+
+Improvements:
+
+- Improved performance when Relational Migrator analyzes large 
+  relational schemas.
+- Relational Migrator now integrates with the Windows certificate store 
+  for SSL connections. 
+
+New features:
+
+- New installer and binaries for Mac M1 architectures.
+- Mapping rules now show the original column and type used to create 
+  an ``_id`` field.
+- Improved error messages in Query Converter (private preview).
```

Comment:
QQ: Should Query Converter be capitalized in this context? It is a feature name and a gui element, I am asking because in another PR I lower cased when not referring to the GUI label specifically and I wanted to double check here :).

Reply:
Thanks for asking! I would lower cases it, when not referring to the GUI label. It would be worth checking with @tomhollander, to ensure that we don't want to make this a proper noun.